File: LP1/Slides2/ex2.8.py
n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))

if (n1 > n2):
    while n1-1>n2:
        n1 = n1 - 1
        print(n1)
elif (n2 > n1):
    while n2-1>n1:
        n2 = n2 - 1
        print(n2)
else:
    print("Os números são iguais")

File: LP1/Slide2/ex2.8.c
#include <stdio.h>
#include <stdlib.h>

int main (void) {
	int n1, n2;
	printf("Digite o primeiro número(inteiro pfvr: \n ");
	scanf("%d", &n1);
	
	printf("Digite o segundo número(inteiro dnv pfvr): \n");
	scanf("%d", &n2);

	if (n1 > n2){
		for (n1; n1>n2+1; n1--){
			printf("%d \n", n1-1);
		}
	}
	if (n2 > n1){
		for(n2; n2>n1+1; n2--){
			printf("%d \n", n2-1);
		}
	} else {
		printf("Os valores sao iguais. \n");
	}
	
}


